src/main/java/com/ericsson/ei/controller/FailedNotificationControllerImpl.java

```diff
@@ -69,11 +67,12 @@
             queryResponse.setQueryResponseEntity(queryResponseEntity);
             LOGGER.debug("The response is : {}", response.toString());
             if (processMissedNotification.deleteMissedNotification(subscriptionName)) {
-                LOGGER.debug("Missed notification with subscription name {} was successfully removed from database", subscriptionName);
+                LOGGER.debug("Failed notification with subscription name {} was successfully removed from database",
+                        subscriptionName);
             }
             return new ResponseEntity<>(queryResponse, HttpStatus.OK);
         } catch (Exception e) {
-            String errorMessage = "Internal Server Error: Failed to extract the data from the Missed Notification Object based on subscription name "
+            String errorMessage = "Internal Server Error: Failed to extract the data from the Failed Notification Object based on subscription name "
```

Comment:
I think we can remove the duplicated "Internal server error" from the log message, since the HTTP status code will anyways return this information to the user. Also why is failed notifications capitalized? They should be lower cased.

Reply:
string "Internal Server Error: " removed